tests/plugins/test_checkpoint_io_plugin.py

```diff
@@ -28,14 +28,14 @@
 
 class CustomCheckpointIO(CheckpointIO):
     save_checkpoint_called: bool = False
-    load_checkpoint_file_called: bool = False
+    load_checkpoint_called: bool = False
```

Comment:
can be removed, unused
the test below uses a mock
forgotten in the original PR :)

Reply:
@ananthsub